src/containers/ThumbnailNavigation.js

```diff
@@ -24,8 +25,22 @@ const mapDispatchToProps = {
   setCanvas: actions.setCanvas,
 };
 
+const styles = {
+  titleBar: {
+    background: 'linear-gradient(to top, rgba(0,0,0,0.7) 0%, rgba(0,0,0,0.3) 70%, rgba(0,0,0,0) 100%)',
+    textAlign: 'left',
+  },
+  title: {
+    color: 'white',
```

Comment:
Is there somewhere I should be getting this color from instead?

Reply:
`theme.palette.common.white`? there's also `theme.palette.getContrastText()`, which I don't understand but sounds promising.

Reply:
I tried this out, but whenever I use `theme` into my styles, everything goes haywire when I switch themes.  Maybe somebody can help me look at this today in the office (or if anybody know of some addl. step I need to do).

Reply:
✋; sounds good